File: backend/src/main/java/com/pofo/backend/domain/admin/login/dto/AdminLogoutResponse.java
package com.pofo.backend.domain.admin.login.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Builder
public class AdminLogoutResponse {
    private String message;
}

File: backend/src/main/java/com/pofo/backend/domain/admin/login/dto/AdminLogoutResponse.kt
package com.pofo.backend.domain.admin.login.dto

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Getter
import lombok.NoArgsConstructor

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Builder
data class AdminLogoutResponse(
    val message: String? = null
)
